### ISSUE TITLE
[Suggestion] How to avoid mining grind in future survival gameplay

### ISSUE THREAD

**Issue by Kaiser-Indrasil** [tsunamayo/Starship-EVO#3186]
Catching up to Discord today, I stumbled upon a conversation about survival mechanics, and mining in particular.
Two main points were that, in practically any voxel building game with a survival mode, mining is a grind.
The other point was, that dumbing down ship acquisition to buying them straigh away takes away an enjoyment of accomplishment.

So, I once again, have to invoke an example of Avorion.
There's grind for resources there, sure, but you only need to dirty your hands yourself for a few hours in the beginning. After that you have mining barges or even carriers to order around to have the resources mined for you. You just have to scout out resource-heavy systems and order your mining fleet to go there and strip-mine, and they will take care of the rest.

And seeing how Stevo already has NPC crew mechanics in mind, I think it would fit neatly into the setting of the game.

Coming back to Avorion, you can also buy ships beside building them from resources, but first off, you have to encounter a shipyard to buy a ship from, as opposed to building one yourself in space. Second, acquiring ships for money instead of resources makes them much more expensive to purchase, around 1.5-2 times more when accounted for the price of materials.

Also, there's rare resource-rich asteroids which incentivise you to build space stations upon, because that way they provide a constant income of the resource the asteroid is filled with.

There's 7 kinds of minerals in Avorion and you can build most blocks from any given one but at the same time, each of them offers a new kind of mechanic. The closer to the galaxy core you get, the better materials you encounter, which in turn let you access more and more advanced gameplay mechanics like shield generators, hyperspace cores, carrier hangars, or hacking computers.

Also, the most advanced mineral - the titular Avorion - is not an be-all end-all building material. It's better to build ship's armour out of trinium because of the best HP-to-mass ratio. Besides, you cannot build heavy plating out of the Avorion mineral anyway.

Sorry for the long post, just wanted to give you some ideas.
Here's the Sass Cat tax (credit goes to Womble):

![image](https://user-images.githubusercontent.com/63756057/98443026-9e086900-2100-11eb-8562-3abe5dd37e9b.png)

**Comment by ultrak2k** [comment]
aye, I agree - having automated planetary long term mining, and space based short term asteroid mining (besides large asteroids I suppose) would be the best choice for this game to not be bog standard simple buy stuff only, and not a grind either

We absolutely need automation with factories and mines, and logistics across systems.

**Comment by tsunamayo** [comment]
Yeah but you need to understand that it will takes time to get automation once I have basic mining, so the game would be a grind for quite some time. Things are much much much more complicated in SEVO compared to Avorion, I wish I have kept building and exploration that simple...

**Comment by JackTSpade** [comment]
@tsunamayo
I'm Jack_Spade, the guy from Discord who doesn't care for the mining grind. I can also see where everyone's coming from, mining resources gives the game some purpose. It gives people a reason to fight, a reason to trade, a reason to explore. Mining doesn't have to be a bad thing- I mostly hate starting out with a hand drill and trying to set up a camp and such. If the player started out at a station with enough money to buy a mining rig, fighter, or something else, it would skip the startup grind and allow the player to decide how he's going to make money. Go mercenary, mine resources, maybe just tag them for other explorers like in Mass Effect. I'm cool with all of it. I just hate the approach Empyrion and Space Engineers took where you spend 100 hours learning to mine so you can eventually never have to do it again. Mining isn't necessarily a bad thing, as long as it isn't a chore; it's just that a lot of games make it a chore.

**Comment by cloudrambler** [comment]
Mining could be fun if the interface works in the players favour. For example; Space Engineers gives you a vague reference on where a large chunk of ore can be found, then penalises you for drilling at the wrong speed when you eventually get to it. That is not fun!

Instead we could have a distinct visual interface that lets the player "see" what is in front of them as they move through the rock. Anyone seen the film "The Core", where they have a density radar map as their main forward display.

The movement through the rock could be stylized, such that a mining ship can progress quickly but not turn well. So digging out small pockets of rock could be quick if you are skilful, or more time consuming if you are careless and miss the target. Thus mining could be a skill reward without being automatically slow.

Also the opportunity for piracy and salvage should be a viable alternative to the stated options of "mine" or "buy".

**Comment by JackTSpade** [comment]
In general, I don't want mining to be complicated. The whole tone of the game is pulpy and bright; it ought to be simple and quick. I still like the idea of tagging ore nodes for someone else to pick up.

**Comment by tsunamayo** [comment]
Yes I think mining have to be beam based at least. It will be similar to salvaging ships. I could add a drill on top of tap for fun, but for gameplay I want something more seamless.

**Comment by Kaiser-Indrasil** [comment]
Yeah, Avorion uses beams for mining and salvage too and it's really convenient to use.
